components/writers/src/main/java/org/datacleaner/extension/output/CreateCsvFileAnalyzer.java

```diff
@@ -88,31 +89,48 @@
     boolean includeHeader = true;
 
     @Configured(order = 6, required = false)
-    InputColumn<?> columnToBeSortedOn ;
+    InputColumn<?> columnToBeSortedOn;
+
+    @Configured
+    boolean overwriteFile;
 
     @Inject
     @Provided
     UserPreferences userPreferences;
 
     private File _targetFile;
-    
-    private int indexOfColumnToBeSortedOn = -1 ;
 
-    private boolean isColumnToBeSortedOnPresentInInput = true ; 
+    private int indexOfColumnToBeSortedOn = -1;
 
+    private boolean isColumnToBeSortedOnPresentInInput = true;
 
     @Initialize
     public void initTempFile() throws Exception {
         if (_targetFile == null) {
-            if(columnToBeSortedOn != null) {
+            if (columnToBeSortedOn != null) {
                 _targetFile = File.createTempFile("csv_file_analyzer", ".csv");
             } else {
-                _targetFile = file ;
+                _targetFile = file;
             }
         }
     }
 
     @Override
+    public String getSuggestedLabel() {
+        if (file == null) {
+            return null;
+        }
+        return file.getName();
+    }
+
+    @Validate
+    public void validate() {
+        if (!file.exists() && overwriteFile == true) {
```

Comment:
I think we should not care about this situation (neither here nor in Excel) - if the file does not exist then we are not running any risks of overwriting existing data. So we should probably also name the configured property like that - instead of "overwriteFile" it should be "overwriteFileIfExists".